### DISCUSSION TITLE
Menu look

### DISCUSSION THREAD

**Discussion by KamiyaMinoru** [Open-Shell/Open-Shell-Menu#863] [Q&A]
![image](https://user-images.githubusercontent.com/78710607/136338557-7c3918ba-8179-43aa-8e07-ce840b8cbeaf.png)

![image](https://user-images.githubusercontent.com/78710607/136338959-8a50048c-9dd6-4439-939f-523e9ec66968.png)

If it is possible to adjust size (width and highness) of menu and font and font size?
I tried to adjust "sub-menu width" and see nothing different
Another option "icon size" will effect lengh of menu unpredictly
Anything elese I can do to customize it? 

Thanks.

**Comment by KamiyaMinoru** [comment]
By the way I had resource hacker
and tried to replace all "fonts" but cause nothing changed seems...
(Or I can not save to apply it currectly)

![image](https://user-images.githubusercontent.com/78710607/136343333-a725a16a-758c-446d-b22a-c4ab0af2c20c.png)

**Comment by Gaurav-Original-ClassicShellTester** [comment]
Look for the Settings "Programs pane width" on the Main Menu tab. There is also a Minimum menu height setting for Windows 7 style which you seem to be using.
